syntax = "proto2";

message l2d_validate_login_response {
	required string key = 1;
	required uint32 result = 2;
	required string gate_addr = 3;
}